subprojects/core-api/src/main/java/org/gradle/process/CommandLineArgumentProvider.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.process;
+
+import org.gradle.api.Incubating;
+
+/**
+ * Provides arguments to a JVM.
+ *
+ * @since 4.6
+ */
+@Incubating
+public interface CommandLineArgumentProvider {
+    /**
+     * The arguments which will be provided to the jvm.
```

Comment:
`jvm` -> `JVM`. Also, we'll be using this type to provide command-line arguments to `Exec` tasks, too, so shall we call it `process` instead?

Alternatively, if this is just about providing arguments to the JVM, then we should call it `JvmCommandLineArgumentProvider` or somesuch.

Reply:
`process` it is.